define.md

```diff
@@ -1,29 +1,18 @@
-# Requirements for full membership
+# Membership Requirements for Member Spaces
 
-## Hackspaces
- A hackspace (also known as Hackerspace, Makerspace, or arts and crafts club for the digital world) is a member run and community operated workspace, where people with similar interests can meet, socialize and collaborate.  See [hackerspace](https://en.wikipedia.org/wiki/Hackerspace) on Wikipedia.
+An organisation is eligible to join the UK Hackspace Foundation as a [Member Space](structure.md) if it:
+    * supports the goals of the UK Hackspace Foundation
+    * is a non-profit association
+    * has the primary aim of providing a shared workshop to its members for the purposes of science, technology, engineering, and art
+    * is democratically controlled by its members, who have equal voting rights and actively participate in policies and making decisions
+    * has a bank account
+    * is based in the UK, Republic of Ireland, or Crown Dependencies
+    * fosters a tolerant and inclusive community
+    * has a clear, non-discriminatory process for members of the public to participate and become members 
+    * does not require affiliation with another organization, such as a university or employer, to be a full member
 
+Applications to join will be decided by the Membership Committee, with appeals decided by the Board.
 
-## Foundation Requirements
-
-* The UK Hackspace Foundation considers an organisation to be eligible to join as a [Member Space](structure.md), if it:
-    * is member owned/operated,
-    * is an unincorporated or incorporated association,
```

Comment:
Did we ditch this in favour of bank account?

Reply:
Discussed in telegram and this was replaced with:

> * is a non-profit association

Along with the requirement for a bank account which automatically requires some degree of legal structure so I'm happy.